components/combobox/filter.js

```diff
@@ -8,9 +8,9 @@ import escapeRegExp from 'lodash.escaperegexp';
 const filter = ({ inputValue, limit = 10, options, selection }) =>
 	options
 		.filter((option) => {
-			const searchTermFound = option.label.match(
-				new RegExp(escapeRegExp(inputValue), 'ig')
-			);
+			const searchTermFound = option.label
```

Comment:
This needs a better experience. I'm not going to block this PR, because I don't think there are SLDS guidelines on the topic of autocomplete outside of global search

<img width="279" alt="screen shot 2018-02-21 at 9 17 15 pm" src="https://user-images.githubusercontent.com/1290832/36520440-37f6e65a-174e-11e8-9cb2-5a3172962bee.png">

Do you have a suggestion? I’m guessing that you’d want the "real" child options under the accounts subheading to show and any options with “acc” in them. (up to the max list item count).

This is the closest guidelines I can find.
https://www.lightningdesignsystem.com/guidelines/search/global

And it seems to say that you shouldn't have more than one subheading when using autocomplete. What do you think about the filter function injecting `{ id: ???, type: 'separator', label:'[Recent items?]' }` or something like that into the filter as the first item when filtering.

Reply:
Hmm, yeah that's a good point. I don't see why they wouldn't want to have more than one sub heading during autocomplete.

Showing that there aren't any "real" child options under subheadings matching the search helps the user understand there wasn't any match under each section, even though the subheading was a match.

For the filter code I have in my wrapper component in lightning report, I do not filter them out.

![screen shot 2018-02-22 at 4 38 05 pm](https://user-images.githubusercontent.com/1821787/36572118-da527f18-17ee-11e8-9a10-9626197d2242.png)

Reply:
I submitted a PR to your branch for the UX that I think Kate R. wants. You don't have to use the filter utilities--we just need to promote good UX patterns in our examples, so engineers can copy them.